egs/voxforge/asr1/local/make_trans.py

```diff
@@ -13,38 +12,40 @@
 if this is the case produces a transcript line for each file in the format:
 prefix_a0405 IT SEEMED THE ORDAINED ORDER OF THINGS THAT DOGS SHOULD WORK
 """
+from __future__ import print_function
 
 import sys
 
 def err(msg):
-    print >> sys.stderr, msg
+    print(msg, file=sys.stderr)
 
 if len(sys.argv) < 3:
     err("Usage: %s <prompts-file> <id-prefix> <utt-id1> <utt-id2> ... " % sys.argv[0])
     sys.exit(1)
 
+#err(str(sys.argv))
 id_prefix = sys.argv[2]
 utt_ids = sys.argv[3:]
 utt2trans = dict()
-unnorm_utt = set()
-
+unnorm_utt = set() 
 for l in file(sys.argv[1]):
     u, trans = l.split(None, 1)
     u = u.strip().split('/')[-1]
-    # convert to upper case
-    trans = unicode(trans, 'utf_8').strip().replace("-", " ").upper()
+    trans = trans.strip().replace("-", " ")
     if not trans.isupper() or \
-       not trans.strip().replace(' ', '').replace("'", "").isalnum():
```

Comment:
Is it OK for this change? (isalnum() vs. isalpha())

Reply:
This file is just copied from kaldi. I'm not sure.